internal/pkg/cli/job_deploy.go

```diff
@@ -356,7 +360,11 @@ func (o *deployJobOpts) manifest() (interface{}, error) {
 	if err != nil {
 		return nil, fmt.Errorf("read job %s manifest: %w", o.name, err)
 	}
-	mft, err := o.unmarshal(raw)
+	interpolated, err := o.newInterpolator(o.appName, o.envName).Interpolate(string(raw))
+	if err != nil {
+		return nil, fmt.Errorf("interpolate environment variables for manifest: %w", err)
```

Comment:
nit: (here and other files)
```suggestion
		return nil, fmt.Errorf("interpolate environment variables for %s manifest: %w", o.name, err)
```